src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,75 @@
+package duke;
+
+import duke.exception.DukeFileSystemException;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * A class that handles anything related to the storing and loading of information from memory for Duke.
+ */
+public class Storage {
+    private final Path filePath;
+    private final DukeParser parser;
+
+    private void createDirectoriesAndFiles() throws IOException {
+        // create storage directories and files if it does not exist
+        if (Files.notExists(filePath)) {
+            File f = new File(filePath.toString());
+            f.getParentFile().mkdirs();
+            f.createNewFile();
+        }
+    }
+
+    /**
+     * Constructs a new instance of Storage that loads and stores data into the file path that is specified.
```

Comment:
There should be a gap between the description and parameters in the javadoc comment.

Reply:
Thanks for spotting these mistakes! I will make the necessary changes :)